photon-lib/src/main/scala/com/linkedin/photon/ml/util/Linalg.scala

```diff
@@ -55,13 +55,38 @@ object Linalg {
     DenseVector(bArr)
   }
 
+  /**
+   * Inverts the matrix $A$, given the Cholesky factorization $l$ of $A$.
+   *
+   * @param l the Cholesky factorization of matrix A
+   * @return the solution
+   */
+  protected[ml] def choleskyInverse(
+      l: DenseMatrix[Double]): DenseMatrix[Double] = {
+
+    val n = l.rows
+
+    // Lapack will write the result to this array
+    val lArr = l.toArray
+
+    val info = new intW(0)
+    lapack.dpotri("L", n, lArr, n, info)
+
+    if (info.`val` < 0) {
+      throw new RuntimeException(
```

Comment:
Can fit on one line

Reply:
IMO it's better for readability to break `if` statements like this into multiple lines, even if they could technically be one-liners.

Reply:
I meant that `throw new RuntimeException(...)` could fit on one line.

I don't think it's right, since it would return an `IllegalArgumentException` and it's not an input, but we could replace the whole `if` with a `require`.